components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -852,6 +853,11 @@ class FennecMigrator private constructor(
                 MigrationFxa.successReason.add(SuccessReasonTelemetryCodes.FXA_SIGNED_IN.code)
                 result
             }
+            is FxaMigrationResult.Success.WillAutoRetrySignInLater -> {
+                logger.debug("Will auto-retry Fennec account migration later")
+                MigrationFxa.successReason.add(SuccessReasonTelemetryCodes.FXA_WILL_RETRY.code)
```

Comment:
Sadly there's no guarantee that the migration will succeed when we retry it later (e.g. maybe you were offline *and* you had a bad sessionToken) but I don't think there's anything we can do about that.

Reply:
That's true. I think counting this as success is fine, since we're deliberate about handling of this situation, and for _most_ people it's probably going to be a success in the end.